cypress/support/pageObjects/LandingPage.ts

```diff
@@ -12,22 +14,19 @@ export default class LandingPage {
     return cy.get(this.headerTitleId);
   }
 
-  private getEarlyAccessButton(): Cypress.Chainable {
-    return cy.get(this.earlyAccessPageButtonId);
+  public getNameErrorMessage(): Cypress.Chainable {
+    return cy.get(this.nameInputErrorId).should('be.visible');
   }
 
-  private retryIfEarlyAccessNotLoaded(attempts: number = 3): void {
-    cy.url().then((currentUrl) => {
-      if (currentUrl !== this.earlyAccessUrl && attempts > 0) {
-        cy.get(this.earlyAccessPageButtonId).click();
-        cy.wait(500);
-        this.retryIfEarlyAccessNotLoaded(attempts - 1);
-      }
-    });
+  public getEmailErrorMessage(): Cypress.Chainable {
+    return cy.get(this.emailInputErrorId).should('be.visible');
```

Comment:
I noticed the assertion `should('be.visible')` was added to the get..ErrorMessage functions. Was this previously implemented elsewhere or just added for extra robustness?

Reply:
Good catch, I added `.should('be.visible')` when I was debugging a bug where Cypress wasn't finding the error messages and left it for extra robustness.